### PR TITLE
[hadd] Turn off multiprocessing if nProcesses is 1

### PR DESCRIPTION
**Pull request by anerokhi**

### CODE REVIEW

**Comment by pcanal** [comment]
@phsft-bot build with flags -DCTEST_TEST_EXCLUDE_NONE=On

**Comment by anerokhi** [comment]
:-/

File: main/src/hadd.cxx [hunk 1]
```diff
@@ -404,6 +404,9 @@ int main( int argc, char **argv )
       std::cout << "Each process should handle at least 3 files for efficiency.";
       std::cout << " Setting the number of processes to: " << nProcesses << std::endl;
    }
+   if (nProcesses == 1)
+      multiproc == kFALSE;
```
Comment (pcanal) [inline main/src/hadd.cxx]:
```suggestion
      multiproc = kFALSE;
```

**Comment by pcanal** [comment]
@phsft-bot build with flags -DCTEST_TEST_EXCLUDE_NONE=On

**Comment by pcanal** [comment]
The failure is unrelated.  Merging.

Thank you very much!

**Pull request merged**